GUI/DeusEx/Classes/MenuChoice_LootActionMelee.uc

```diff
@@ -0,0 +1,14 @@
+class MenuChoice_LootActionMelee extends MenuChoice_LootAction;
+#compileif injections
+
+defaultproperties
+{
+    HelpText="Should melee weapons be included in the list of Junk items that get dropped when looting a body?"
+    actionText="Melee Weapons"
```

Comment:
I think melee weapons should be refused by default? it's probably time we get rid of the awkward weapon dropping when dying thing and instead lean on this system

Reply:
If we do that, maybe the drop-when-dying option should be removed.

Reply:
yes, maybe not removed but disabled by default at least

Reply:
What purpose would it still serve? People either want to loot melee weapons from corpses or they don't.

Reply:
people may be comfortable with the old method, it does give visibility of what weapon it is, like if you're looking for a sword but not looting every body

Reply:
How much time do you spend looking for a specific melee weapon (I mean, besides the obvious one)? Isn't that kind of the point here, that you quickly stop wanting to be bothered by them?